content/en/docs/reference/config-api/kubelet-config.v1beta1.md

```diff
@@ -645,7 +655,7 @@ Default: true</p>
 </td>
 <td>
    <p>cpuCFSQuotaPeriod is the CPU CFS quota period value, <code>cpu.cfs_period_us</code>.
-The value must be between 1 us and 1 second, inclusive.
+The value must be between 1 ms and 1 second, inclusive.
 Requires the CustomCPUCFSQuotaPeriod feature gate to be enabled.
 Default: &quot;100ms&quot;</p>
```

Comment:
Aside: this feels a bit off (something to consider fixing _upstream_).

Reply:
This appears to have been fixed in Kubernetes v1.26.
```
clarify CPUCFSQuotaPeriod values, set the minimum to 1ms
```
https://github.com/kubernetes/kubernetes/commit/d0f9e6dc36fb0f6cfff95988e27eb3796c4e6bce

Reply:
@mengjiao-liu So you mean the generated documentation is up to date, right?

Reply:
> @mengjiao-liu So you mean the generated documentation is up to date, right?

I mean, upstream fixed this issue in version 1.26, but the auto-generated files are still unfixed, and I'm not sure if the auto-generated files are up to date or if there are other issues.